task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -57,32 +63,35 @@ spec:
         check_num=4
         failure_num=0
         TESTPASSED=true
-        chmod +x opm grpc_health_probe
 
-        if ! ./opm version; then
-          echo "!FAILURE! - opm binary check failed"
+        if [ ! -L "bin/opm" ]; then
+          echo "!FAILURE! - opm binary presence check failed"
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
-        if [ ! -f "grpc_health_probe" ]; then
+        if [ ! -f "usr/bin/grpc_health_probe" ]; then
           echo "!FAILURE! - grpc_health_probe binary check failed"
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
-        if ! ./opm validate confdir; then
+        if ! opm validate ."${conffolder}"; then
           echo "!FAILURE! - opm validate check has failed"
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
-        if ! ./opm render confdir | jq -en 'reduce (inputs | select(.schema == "olm.package")) as $obj (0; .+1) == 1'; then
+        if ! opm render ."${conffolder}" | jq -en 'reduce (inputs | select(.schema == "olm.package")) as $obj (0; .+1) == 1'; then
           echo "!FAILURE! - more than one olm.packages are not permitted in a FBC fragment"
           failure_num=`expr $failure_num + 1`
           TESTPASSED=false
         fi
         if [ $TESTPASSED == false ]; then
           HACBS_ERROR_OUTPUT="$(make_result_json -r FAILURE -f $failure_num -s `expr $check_num - $failure_num`)"
           echo "${HACBS_ERROR_OUTPUT}" | tee $(results.HACBS_TEST_OUTPUT.path)
+          cd ..
```

Comment:
We may delete the file out of the  `if..else..` so we can have only one `cd & rm -fr`, but it is not a big deal.

Reply:
Thats a good point. Thanks!